index.html

```diff
@@ -395,7 +395,7 @@ <h2>The <dfn>primary pointer</dfn></h2>
             <section>
                 <h2>Firing events using the <code>PointerEvent</code> interface</h2>
                 <p>To <dfn>fire a pointer event</dfn> named |e| means to [=fire an event=] named |e| using <a>PointerEvent</a> whose attributes are set as defined in {{PointerEvent}} Interface and <a>Attributes and Default Actions</a>.</p>
-                <p>If the event is not {{GlobalEventHandlers/gotpointercapture}} or {{GlobalEventHandlers/lostpointercapture}}, run the <a>process pending pointer capture</a> steps for this <code>PointerEvent</code>.
+                <p>If the event is not a {{GlobalEventHandlers/gotpointercapture}}, {{GlobalEventHandlers/lostpointercapture}}, <code>click</code>, <code>auxclick</code> or <code>contextmenu</code> event, run the <a>process pending pointer capture</a> steps for this <code>PointerEvent</code>.
```

Comment:
Given that we want to release pointer capture before click, auxclick or contextmenu, shoudln't we keep this here?

Reply:
I think excluding these events here is reasonable for a few reasons:
- As per the "event dispatch" subsection we added in this PR, the dispatch for click, auxclick and contextmenu is completely separate from all other PE types.
- Because this section implicitly covers PEs that are fired from non-pointer devices (such as keyboard click), we need to isolate these 3 events anyway.  Before this PR, the isolation was very vague (to be precise, "run the process pending pointer capture steps **for this PointerEvent**" doesn't apply because keyboard click has id=-1).
- Pending pointer capture gets resolved before we fire any of these 3 events I believe.

Do you see a scenario where the last point does not hold?

Reply:
Landing it now, we can cover corner cases separately.